app/controllers/obs_controller.rb

```diff
@@ -2,10 +2,19 @@
 
 class OBSError < StandardError; end
 
+HTTP_ERRORS = [
+  EOFError,
+  Errno::ECONNRESET,
+  Errno::EINVAL,
+  Net::HTTPBadResponse,
+  Net::HTTPHeaderSyntaxError,
+  Net::ProtocolError,
+  Timeout::Error
+].freeze
+
 # Handle connection to OBS
 class OBSController < ApplicationController
-  before_action :set_distributions
-  before_action :set_releases_parameters
+  before_action :set_baseproject # depends on set_releases_parameters and set_distributions
```

Comment:
I'd remove the comment here as the dependencies are clear inside the method and it does not really add much clarity to developers (and it is another place one has to change when changing set_baseproject which, if forgotten, is confusing)